src/index.js

```diff
@@ -29,36 +29,54 @@ import './images/step-icon.png'
 import './images/trend-icon.png'
 import './images/water-icon.png'
 
+let currentUserID;
 
-  let userData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData').then(response => response.json()).then(json => json.userData)
-  // console.log(userData)
-  let sleepData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/sleep/sleepData').then(response => response.json()).then(json => json.sleepData)
-  let hydrationData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/hydration/hydrationData').then(response => response.json()).then(json => json.hydrationData)
-  let activityData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/activity/activityData').then(response => response.json()).then(json => json.activityData)
+$('nav').hide()
+$('header').hide()
+$('main').hide()
 
-  let randomId, stats, userRepository, hydrationRepository, sleepRepository, activityRepository, user;
+$('.splash__input--user').keyup((e) => {
+  e.preventDefault();
+  if ($('.splash__input--user').val() !== '') {
+    $('.splash__button').prop('disabled', false);
+  }
+})
 
- Promise.all([userData, sleepData, hydrationData, activityData])
-  .then(data => {
-    userData = data[0];
-    sleepData = data[1];
-    hydrationData = data[2];
-    activityData = data[3];
-  })
+$('.splash__button').on('click', (e) => {
+  var currentUserID = $('.splash__input--user').val();
```

Comment:
@trishalanglois Have you tried `currentUserID = $('.splash__input--user').val();`? The way it is now, you might just be creating a new local variable the same name rather than reassigning.

Reply:
Yeah, I tried that, too, and it's still not working.  I just tried to hard-code in a number into the .val(), also, and that isn't working either.  The DOM is just showing up without any of the dynamic user data.